src/com/company/Main.java

```diff
@@ -37,26 +39,38 @@ public static void play() {
         shouldPlayAgain = true;
         player = PlayerSignType.O;
 
-        final int boardSize = getPlayingFieldSize(MIN_BOARD_SIZE, MAX_BOARD_SIZE);
-        System.out.printf(FORMATED_SELECT, SELECTED, boardSize);
+        ioManager.showMessage(PROVIDE_YOUR_NAME);
+        final String name = ioManager.getName();
+        ioManager.showMessage(format(FORMATED_WELCOME, HELLO, name));
+
+        ioManager.showMessage(PROVIDE_BOARD_SIZE_TEXT);
+        final int boardSize = getNumberFromUser(MIN_BOARD_SIZE, MAX_BOARD_SIZE);
+        ioManager.showMessage(format(FORMATED_SELECT, SELECTED, boardSize));
 
         final Board board = new Board(boardSize);
 
+        player = changePlayer(player);
```

Comment:
Czemu to wywędrowało przed pętlę?

Reply:
Bo nie może się wykonywać zawsze. Jeśli na przykład wybierzesz już zajęte miejsce na planszy, to dostaniesz komunikat o błędzie, ale nadal gra ten sam gracz, nie może się zmienić. A na początku wykonywał się zawsze. W sumie to można to wyrzucić (linie przed pętlą) bo zmienna jest zainicjalizowana. A jak dorobię drugiego gracza to warto by było losować pierwsze przyporządkowanie, żeby nie było zawsze jednakowo (i nudno).

Reply:
linia wyrzucona